src/main/java/view/InputViewImpl.java

```diff
@@ -0,0 +1,37 @@
+package view;
+
+import domain.Name;
+import domain.TryCount;
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputViewImpl implements InputView {
```

Comment:
```suggestion
public class ConsoleInputView implements InputView {
```

`-Impl` suffix는 deprecated되어 가는 것 같아서, 네이밍 하나 제안드립니다 !

Reply:
관습적으로 -impl을 썼는데 이에 대해 비판하는 글들이 많네요!